app/views/collections/_form.html.erb

```diff
@@ -1,9 +1,19 @@
 <%= simple_form_for @form, html: { class: 'editor' } do |f| %>
   <div id="descriptions_display">
-    <h2 class="non lower">Descriptions</h2>
-    <% f.object.rendered_terms.each do |term| %>
-      <%= render_edit_field_partial term, f: f %>
-    <% end %>
+    <h2 class="non lower">Metadata</h2>
```

Comment:
I believe the Sufia UI WG asked for this header to say `Descriptions` rather than `Metadata`. Tagging @projecthydra/sufia-ui-ux-advisors 

(In which case, we'll want our forms and presenters to standardize on `Descriptions`.)